### DISCUSSION TITLE
Possible to preserve last search ?

### DISCUSSION THREAD

**Discussion by ghost** [tomrijndorp/vscode-finditfaster#27] [Q&A] [answered]
Hi, great work. 
Is is possible to preserve the view from last search ?
When i hit enter the terminal closes and takes me to the file. But I want to come back and check other results.

**Comment by tomrijndorp** [comment, marked as answer]
There's a perhaps better way to do that, actually :) You can type `tab` to select files. When you then press `enter`, all the files you selected are opened.
